integration-tests/test-api/src/index.ts

```diff
@@ -69,15 +69,18 @@ app.all("/file-upload/:id/:status{[0-9]{3}}", async (c) => {
 
 app.all("/get-stats/:id", (c) => {
   const id = c.req.param("id");
+  console.log("getting stats", id);
```

Comment:
do you still want to include these console logs?

Reply:
These are handy when testing locally making sure that you can keep track of what's going on, so I think I'll leave them in.

Reply:
there might be some value in abstracting this out to a logging util that is controlled by an environmental variable, so you don't pollute logs by default, but you can opt-into debug mode.

logging util will also allow you to add a common prefix, which I think is nice here.

Reply:
It doesn't seem like GitHub Action outputs these logs anyways, so the workflow doesn't seem to get as clogged, though it would be nice in the future to maybe give this little API a bit more love.